### DISCUSSION TITLE
MIMIC-IV 2.0 in Google BigQuery

### DISCUSSION THREAD

**Discussion by XiangBu** [MIT-LCP/mimic-code#1320] [MIMIC-IV] [answered]
hi, will you provide MIMIC-IV 2.0 in Google BigQuery recently,Thx!

**Comment by alistairewj** [comment]
Yes will do! When we make a new release of the mimic-code repo, that will indicate that BigQuery is updated.

**Reply by XiangBu** [reply]
ok, GOT IT! THX a lot!

**Comment by alistairewj** [comment, marked as answer]
MIMIC-IV v2.0 is available in two BigQuery datasets under the physionet-data project: `mimiciv_hosp` and `mimiciv_icu`. MIMIC-IV-ED v2.0 is available in `mimiciv_ed`. You will need to re-sign the DUA on PhysioNet and re-acquire BigQuery access through PhysioNet to see the data. If it still doesn't appear, try a hard refresh on the BigQuery page: Ctrl/Cmd + Shift + R.

We will be keeping MIMIC-IV v1.0 and MIMIC-IV-ED v1.0 available at the original location (`mimic_core`, `mimic_hosp`, `mimic_icu`, and `mimic_ed`) until July 31st 2022 to give users some time to transition to the new version. After that, we will move these to an archive project pending deprecation.

**Comment by tompollard** [comment]
See https://github.com/MIT-LCP/mimic-code/discussions/1154 for more information on how to connect to MIMIC (and other PhysioNet datasets) on Google Cloud/BigQuery.

**Comment by tompollard** [comment]
The process for pinning a project on BigQuery has recently changed. See: https://github.com/MIT-LCP/mimic-code/discussions/1453 for a step-by-step description of the new process.
